tests/mlio-py-test/test_data_reader_params.py

```diff
@@ -26,6 +26,22 @@ def test_recordio_protobuf_reader_params():
     assert reader2.peek_example()
 
 
+def test_text_line_reader_params():
+    expected_string = "this is line 1"
+    filename = os.path.join(resources_dir, 'test.txt')
+    dataset = [mlio.File(filename)]
+    rdr_prm = mlio.DataReaderParams(dataset=dataset,
+                                    batch_size=1)
+
+    reader = mlio.TextLineReader(rdr_prm)
+    example = reader.read_example()
+    record = [as_numpy(feature) for feature in example]
+    reader2 = mlio.TextLineReader(rdr_prm)
+
+    assert record[0].squeeze() == expected_string
+    assert reader2.peek_example()
```

Comment:
What exactly does this assertion verify? Are we not already testing the same condition implicitly via the first reader's assertion?

Reply:
yes, agreed. it is being implicitly tested.